wis2downloader/Dockerfile

```diff
@@ -18,7 +18,7 @@ ENV LOG_PATH /app/logs
 ENV WIS2DOWNLOADER_CONFIG "/app/config.json"
 
 # update pyopenssl and pin requests and urllib3 to avoid SSL error
-RUN pip install pyopenssl --upgrade && pip install requests==2.26.0 urllib3==1.26.0
```

Comment:
@david-i-berry can we have a Docker setup in wis2downloader proper?  This would separate concerns and allow usage of wis2downloader in Docker workflow in standalone mode.

Reply:
@tomkralidis what do you mean by "use of wis2downloader in Docker workflow in standalone mode"? At the moment with the wis2box integration we rely on the wis2box / nginx auth, in standalone mode we'd want to revisit this.